e2e/cypress/integration/pages/account/add-to-order-template.module.ts

```diff
@@ -0,0 +1,34 @@
+export class AddToOrderTemplateModule {
```

Comment:
There are still e2e tests pending analogous to the wishlist management and shopping experience tests